src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -2650,6 +2654,20 @@ SpinelNCPInstance::property_set_value(
 				.finish()
 			);
 
+		} else if (strcaseequal(key.c_str(), kWPANTUNDProperty_ThreadChildTimeout)) {
+			uint32_t child_timeout = any_to_int(value);
+			Data command = SpinelPackData(SPINEL_FRAME_PACK_CMD_PROP_VALUE_SET(SPINEL_DATATYPE_UINT32_S), SPINEL_PROP_THREAD_CHILD_TIMEOUT, child_timeout);
+
+			mSettings[kWPANTUNDProperty_ThreadChildTimeout] = SettingsEntry(command);
+
+			start_new_task(SpinelNCPTaskSendCommand::Factory(this)
+				.set_callback(cb)
+				.add_command(
+					SpinelPackData(SPINEL_FRAME_PACK_CMD_PROP_VALUE_SET(SPINEL_DATATYPE_UINT32_S), SPINEL_PROP_THREAD_CHILD_TIMEOUT, child_timeout)
```

Comment:
We can use `command` variable here since we create spinel frame and place in this variable above. No need to re-create the spinel frame again.